diesel/src/macros/mod.rs

```diff
@@ -177,6 +177,23 @@ macro_rules! __diesel_column {
 /// # }
 /// ```
 ///
+/// If you are using types that aren't from Diesel's core types, you can specify
+/// which types to import. Note that the path given has to be an absolute path
+/// relative to the crate root. You cannot use `self` or `super`.
+///
+/// ```ignore
+/// #[macro_use] extern crate diesel;
+/// extern crate diesel_full_text_search;
```

Comment:
Probably, but I want to document a real use case here. The doctest wouldn't add anything useful over the existing tests (other than ensuring the example works of course, but meh...)